src/components/film-list.js

```diff
@@ -1,34 +1,49 @@
-import {getFilmCardTemplate} from './film-card.js';
-import {getButtonShowTemplate} from './button-show-more.js';
+import {getFilmCardTemplate} from './film-card';
+import {getButtonShowTemplate} from './button-show-more';
```

Comment:
Как вариант:
```suggestion
import {getLoadButtonTemplate} from './load-button;
```

Вообще с названием беда 😟